lib/inv_eigcg_quda.cpp

```diff
@@ -16,10 +16,11 @@
 #include <iostream>
 
 #include <blas_magma.h>
+#include <Eigen/Dense>
```

Comment:
I think eigen is not activated by default. We usually guard includes of eigen with #ifdefs. We might need to discuss whether we make eigen a dependency of QUDA by default. I like eigen but the downside is that we cannot ship it with QUDA and making it a default thing will require an additional dependency step in building QUDA.